src/components/complete-profile-second/CompleteProfileSecond.tsx

```diff
@@ -0,0 +1,71 @@
+import { memo, useState } from 'react';
+import { useLocales } from 'src/locales';
+import WorkForm from './work-form/WorkForm';
+import WorkList from './work-list/WorkList';
+import { ProfileExperience } from './types';
+import { Title } from './styles';
+
```

Comment:
Would be better if imports were absolute(not sure whether it's a problem but it's in pr template)

Reply:
fixed